Zinc/Private/ZincRepo.m

```diff
@@ -1297,15 +1297,12 @@ - (BOOL) doesPolicyAllowDownloadForBundleID:(NSString*)bundleID
 
 - (void) logEvent:(ZincEvent*)event
 {
-    __weak typeof(self) weakself = self;
-    [[NSOperationQueue mainQueue] addOperationWithBlock:^{
-        __strong typeof(weakself) strongself = weakself;
-        if ([strongself.eventListener respondsToSelector:@selector(zincRepo:didReceiveEvent:)])
-            [strongself.eventListener zincRepo:strongself didReceiveEvent:event];
-        
-        NSMutableDictionary* userInfo = [event.attributes mutableCopy];
-        [[NSNotificationCenter defaultCenter] postNotificationName:[[event class] notificationName] object:self userInfo:userInfo];
-    }];
+    if ([self.eventListener respondsToSelector:@selector(zincRepo:didReceiveEvent:)]) {
+        [self.eventListener zincRepo:self didReceiveEvent:event];
+    }
+
+    NSMutableDictionary* userInfo = [event.attributes mutableCopy];
+    [self postNotification:[[event class] notificationName] userInfo:userInfo];
```

Comment:
`postNotification:userInfo:` does the same jump to the main queue for posting the notification: https://github.com/mindsnacks/Zinc-ObjC/blob/ddec7d1da29b9d92d6456f9596ff295ff5437283/Zinc/Private/ZincRepo.m#L249-L258